4.6.2/WcfBasicIISHost/WcfServiceTestClient/Service References/ServiceReference1/Reference.svcmap

```diff
@@ -18,14 +18,14 @@
     <ServiceContractMappings />
   </ClientOptions>
   <MetadataSources>
-    <MetadataSource Address="http://localhost/wcfservicetest/Service1.svc" Protocol="http" SourceId="1" />
+    <MetadataSource Address="http://10.0.75.1:84/Service1.svc" Protocol="http" SourceId="1" />
```

Comment:
Good catch! Customer needs to reply the real host name anyway, I will replace the IP address with placeholder host name